### ISSUE TITLE
Questions about the change of ReID model

### ISSUE THREAD

**Issue by corentin87** [phil-bergmann/tracking_wo_bnw#154]
Hi,
I was reading your paper and saw at that time, you were using a Siamese Network and getting a similarity score. But digging a bit into your repo I saw the commit when you switched to Torchreid  and Resnet-50 to get features of an image and then measure the distance between two vectors.
Why did you stop using your Siamese CNN? Was ReID with Resnet-50 more accurate and faster?
Thank you

**Comment by timmeinhardt** [comment]
Hello, we are still using the concept of a Siamese network. It just means applying the same network to extract features from two inputs and then we compare their distance. The switch to torchreid did not change this as we were also using a ResNet50 before.

**Comment by corentin87** [comment]
Ok thanks so you still train two ResNet50 as a Siamese Network. What was your idea to update your first implementation to TorchReid? Just easier to use? Did you retrain it compare to the model available in the Model-Zoo of Torchreid?

**Comment by timmeinhardt** [comment]
A Siamese network contains two identical subnetworks. This means in practice you only train one ResNet50 but apply it twice/in parallel.

**Comment by corentin87** [comment]
Yes got it. Also last question your resnet50-fc512 trained on Market1501 using TorchReiD is 300MB while the same network in the TorchReid Model Zoo is only 100MB. Why is that?
Thanks.

**Comment by timmeinhardt** [comment]
Our model file includes the state dictionary of the model, optimizer and learning rate scheduler. I assume the models from the official model zoo only include the first.

**Comment by corentin87** [comment]
ok thanks for your replies!

**Comment by timmeinhardt** [comment]
No worries. Can we close this issue?
